### PR TITLE
Pokedex Triggers

### PR DESCRIPTION
**Pull request by sdougbrown**
This caught my fancy.  I thought we could do a better job here, so this is just a start. :smile: 

to @moollaza 
cc @jagtalon

### CODE REVIEW

File: lib/DDG/Spice/Pokemon/Data.pm [hunk 1]
```diff
@@ -17,7 +17,8 @@ code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/
 attribution github => ["AkA84"];
 
 # Triggers
-triggers startend => "pokemon";
+triggers startend => "pokemon", "pokedex";
+triggers end => "evolution";
```
Comment (iambibhas) [inline lib/DDG/Spice/Pokemon/Data.pm]: This would make it trigger for `human evolution` or similar too, right?

Comment (jagtalon) [inline lib/DDG/Spice/Pokemon/Data.pm]: @iambibhas there are no human pokemons so it should be ok, right?

Comment (iambibhas) [inline lib/DDG/Spice/Pokemon/Data.pm]: No. :P But it'd still trigger and make an API call, right?

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**